Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -291,6 +291,22 @@ public static void DismissAlert(this UITestContext context)
         context.Driver.SwitchTo().DefaultContent();
     }
 
+    /// <summary>
+    /// Clicks on the first matching element, switches control to JS alert/prompt box, if <paramref name="inputText"/>
+    /// is not <see langword="null"/> enters it as keystrokes, accepts the alert/prompt box, and switches control back
+    /// to main document or first frame.
```

Comment:
I've allowed myself to clarify a bit:
```suggestion
    /// Clicks on the first matching element, switches control to the JS alert/prompt box that's expected to appear,
    /// enters <paramref name="inputText"/> as keystrokes if it's not <see langword="null"/>, accepts the alert/prompt
    /// box, and switches control back to main document or first frame.
```